### DISCUSSION TITLE
Can't seem to setup email. Keep getting configuration error.

### DISCUSSION THREAD

**Discussion by ahmetskilinc** [payloadcms/payload#727] [Q&A]
Keep recieving below error whenever I run payload:
`ERROR (payload): There is an error with the email configuration you have provided.`

server.js
```javascript
// ...
const nodemailer = require("nodemailer");
// ...
let transport = nodemailer.createTransport({
	service: "iCloud",
	auth: {
		user: process.env.SMTP_USER,
		pass: process.env.SMTP_PASS,
	},
});

payload.init({
    // ...
	email: {
		fromName: "Ahmet",
		fromAddress: "noreply@email.com",
		transport,
	},
});

app.listen(3000);
```

Are there any glaring issues?

**Comment by DanRibbens** [comment]
I'm not familiar with using iCloud for sending transactional mail. Payload uses nodemailer which uses SMTP. You'll need to find setup steps for allowing SMTP for your iCloud account.

My guess is that you can't use the `fromAddress` as `noreply@email.com`. Are you using your own email in that field?
